packages/universal/reactive/README.md

```diff
@@ -53,6 +56,8 @@ stored elsewhere. For example, reactive collections store their values in the
 JavaScript collections they represent, and use markers to represent each
 discrete piece of storage in the collection.
 
+> You can think of a marker as a kind of cell without a value.
```

Comment:
Why maker instead of ref? (given the above description: "represents values stored elsewhere")?

Reply:
is because it's just a representation, and doesn't actually reference anything?